python/heterocl/tvm/build_module.py

```diff
@@ -502,10 +507,13 @@ def build_fpga_kernel(sch, args, target, name="default_function", schedule_name=
             else:
                 vals.insert(3, "")
             keys.insert(4, "project")
-            if schedule_name != "":
-                folder = "{}-{}".format(schedule_name,target.project)
-            else:
-                folder = target.project
+            # temporarily removed

```

Comment:
Can you add options for users to choose whether they want to run their code in the same project?

Reply:
Could you elaborate on "running the code in the same project"?

Reply:
IMO we best add that in another PR, just to keep things separate.